repokid/repokid.py

```diff
@@ -271,15 +271,15 @@ def display_roles(account_number, inactive=False):
 
 def _get_role_permissions(role):
     permissions = set()
-    for policy_name, policy in _get_current_policies(role).items():
+    for policy_name, policy in role.policies[-1]['Policy'].items():
```

Comment:
Retrieving the last version of the policy (latest) because we are storing previous versions of policies in case we need to roll back.